#! /bin/sh /usr/share/dpatch/dpatch-run
# Description: Disable some failing tests

@DPATCH@
--- mysql-cluster-7.2.4.orig/mysql-test/suite/ndb/t/disabled.def	2012-03-06 18:29:31.000000000 +0000
+++ mysql-cluster-7.2.4/mysql-test/suite/ndb/t/disabled.def	2012-03-06 18:31:55.000000000 +0000
@@ -19,3 +19,6 @@
 ndb_sql_allow_batching   : SEAGULL WL3733 xxx_allow_batching
 
 ndb_dbug_lock : SEAGULL gsl locking order
+
+ndb.ndb_join_pushdown_default : Fails on 3rd pass with memory error
+
